### PR TITLE
Make sure email recipients can see issue

### PR DESCRIPTION
**Pull request by jolheiser**
As title

### CODE REVIEW

**Review by 6543 — APPROVED** [review]
:+1:

File: services/mailer/mail_issue.go [hunk 1]
```diff
@@ -122,7 +122,17 @@ func mailIssueCommentBatch(ctx *mailCommentContext, ids []int64, visited map[int
 		if err != nil {
 			return err
 		}
-		// TODO: Check issue visibility for each user
+
+		// Make sure all recipients can still see the issue
+		idx := 0
+		for _, r := range recipients {
+			if ctx.Issue.Repo.CheckUnitUser(r, models.UnitTypeIssues) {
```
Comment (zeripath) [inline services/mailer/mail_issue.go:129]: this could be very expensive as currently constituted but it does solve the issue.

**Review by zeripath — APPROVED** [review]

**Comment by jolheiser** [comment]
Ping lgtm

**Comment by jolheiser** [comment]
@jolheiser please send backport 🙃

**Pull request merged**